policyengine_us/tests/policy/baseline/gov/states/ms/tax/income/deductions/itemized/ms_itemized_deduction.yaml

```diff
@@ -0,0 +1,47 @@
+- name: MS itemized deduction unit test 1
+  absolute_error_margin: 0.01
+  period: 2021
+  input:
+    state_code: MS
+    adjusted_gross_income: 100_000
+    charitable_deduction: 100
+    medical_expense_deduction: 1_000
+    misc_deduction: 0
+  output:
+    ms_itemized_deduction: 1_100
+
+- name: MS itemized deduction unit test 2
+  absolute_error_margin: 0.01
+  period: 2021
+  input:
```

Comment:
you can use `itemized_deductions_less_salt` instead of medical etc. for the unit test we only try to use input variables from the formula, even if both would work